### PR TITLE
	remove morden add ropsten

### PR DESCRIPTION
**Pull request by winsvega**
ropsten block
was there eip150 block for morden?

### CODE REVIEW

File: libethashseal/GenesisInfo.cpp [hunk 1]
```diff
@@ -88,8 +91,7 @@ R"ETHEREUM(
 		"durationLimit": "0x0d",
 		"blockReward": "0x4563918244F40000",
 		"registrar": "",
-		"networkID" : "0x02",
-		"chainID": "0x02"
+		"networkID" : "0x02""
```
Comment (chriseth) [inline libethashseal/GenesisInfo.cpp]: Double double quote.

File: libethashseal/genesis/olympic.h [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+using namespace dev;
+using namespace eth;
+
+namespace dev
+{
+namespace eth
+{
+static h256 const c_genesisStateRootOlympic;
+static std::string const c_genesisInfoOlympic =
+R"ETHEREUM(
```
Comment (chfast) [inline libethashseal/genesis/olympic.h]: Why this is in a header??? You will have a copy of it in every compilation unit including the header.

Comment (winsvega) [inline libethashseal/genesis/olympic.h]: pragma once wont help? 
how else could I split this variables into different files?

Comment (gcolvin) [inline libethashseal/genesis/olympic.h]: If you only need them in genesisinfo.cpp why not just put them there?

File: eth/main.cpp [hunk 3]
```diff
@@ -634,6 +635,8 @@ int main(int argc, char** argv)
 			chainParams = ChainParams(genesisInfo(eth::Network::Olympic));
 		else if (arg == "--morden" || arg == "--testnet")
```
Comment (chriseth) [inline eth/main.cpp]: I think it is a good idea to move `--testnet` to `--ropsten`.

File: libethashseal/genesis/GenesisInfo.cpp [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		You should have received a copy of the GNU General Public License
+		along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file GenesisInfo.cpp
+ * @author Gav Wood <i@gavwood.com>
+ * @date 2014
+ */
+
+#include "../GenesisInfo.h"
+#include "eip150test.cpp"
```
Comment (chfast) [inline libethashseal/genesis/GenesisInfo.cpp]: Here place only: `extern std::string const c_genesisInfoEIP150Test;` (not `#include`).

File: libethashseal/genesis/eip150test.cpp [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+ * @author Dimitry Khokhlov <winsvega@mail.ru>
+ * @date 2016
+ */
+
+#include "../GenesisInfo.h"
+using namespace dev;
+using namespace eth;
+
+static h256 const c_genesisStateRootEIP150Test;
+static std::string const c_genesisInfoEIP150Test = std::string() +
```
Comment (chfast) [inline libethashseal/genesis/eip150test.cpp]: In this file keep only `std::string const c_genesisInfoEIP150Test(R"...")`, and state root hash if needed.

File: libethashseal/genesis/eip150test.cpp [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+		cpp-ethereum is distributed in the hope that it will be useful,
+		but WITHOUT ANY WARRANTY; without even the implied warranty of
+		MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+		GNU General Public License for more details.
+
+		You should have received a copy of the GNU General Public License
+		along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file eip150test.cpp
+ * @author Dimitry Khokhlov <winsvega@mail.ru>
```
Comment (chfast) [inline libethashseal/genesis/eip150test.cpp]: @chriseth can we remove these tags. They are very redundant and do not provide true information.

File: libethashseal/genesis/GenesisInfo.cpp (hunk 4 above)
Comment (winsvega) [inline libethashseal/genesis/GenesisInfo.cpp]: would not work. tried

File: libethashseal/genesis/eip150test.cpp (hunk 6 above)
Comment (winsvega) [inline libethashseal/genesis/eip150test.cpp]: remove all the GavinWood author tags?  ))

File: eth/main.cpp (hunk 3 above)
Comment (winsvega) [inline eth/main.cpp]: done

File: eth/main.cpp [hunk 7]
```diff
@@ -907,7 +904,7 @@ int main(int argc, char** argv)
 		catch (...)
 		{
 			cerr << "provided configuration is not well formatted" << endl;
-			cerr << "sample: " << endl << genesisInfo(eth::Network::Olympic) << endl;
+			cerr << "sample: " << endl << genesisInfo(eth::Network::Frontier) << endl;
```
Comment (chriseth) [inline eth/main.cpp]: Isn't that string gigantic (it includes all the presale info)? Please use ropsten.

File: libethashseal/CMakeLists.txt [hunk 8]
```diff
@@ -1,6 +1,18 @@
+
+macro (add_sources)
```
Comment (chriseth) [inline libethashseal/CMakeLists.txt]: Why is this cmake magic needed?

**Pull request merged**